scheds/rust/scx_rustland/src/main.rs

```diff
@@ -247,13 +247,13 @@ impl<'a> Scheduler<'a> {
     // This method implements the main task ordering logic of the scheduler.
     fn update_enqueued(&mut self, task: &mut QueuedTask) -> u64 {
         // Update global minimum vruntime based on the previous task's vruntime.
-        if self.min_vruntime < task.vtime {
+        if self.min_vruntime > task.vtime {
```

Comment:
Actually we want to move min_vruntime forward.

You can see `self.min_vruntime`  as a clock that is always moving forward, and the "current time" is determined by the task that is enqueued. Ideally we would like to update the `min_vruntime` using the vtime of the task that is running on a CPU, but we don't have direct callbacks from ops.running() in the Rust part of the scheduler, so this is like an approximation.

Maybe we should clarify better this concept renaming `self.min_vruntime` to `self.vruntime_now` or something similar...